### DISCUSSION TITLE
Farming status: Not synced or not connected to peers

### DISCUSSION THREAD

**Discussion by Thanatos-JFJ** [Chia-Network/chia-blockchain#17335] [Support]
ubuntu 18.04
chia version: 2.1.4rc3.dev195

Farming status: Not synced or not connected to peers
Total chia farmed: 1642.109158461903
User transaction fees: 0.109158461903
Block rewards: 1642.0
Last height farmed: 4753253
Local Harvester
   0 plots of size: 0.000 MiB on-disk, 0.000 MiBe (effective)
Remote Harvester for IP: 192.168.5.52
   3246 plots of size: 321.304 TiB on-disk, 321.430 TiBe (effective)
Remote Harvester for IP: 192.168.5.49
   2961 plots of size: 293.089 TiB on-disk, 293.208 TiBe (effective)
Remote Harvester for IP: 192.168.5.55
   2740 plots of size: 271.211 TiB on-disk, 271.324 TiBe (effective)
Remote Harvester for IP: 192.168.5.57
   1285 plots of size: 127.194 TiB on-disk, 127.245 TiBe (effective)
Remote Harvester for IP: 192.168.5.50
   2590 plots of size: 256.369 TiB on-disk, 256.471 TiBe (effective)
Remote Harvester for IP: 192.168.5.46
   2700 plots of size: 267.247 TiB on-disk, 267.363 TiBe (effective)
Remote Harvester for IP: 192.168.5.53
   2801 plots of size: 277.258 TiB on-disk, 277.365 TiBe (effective)
Remote Harvester for IP: 192.168.5.56
   2721 plots of size: 269.336 TiB on-disk, 269.443 TiBe (effective)
Remote Harvester for IP: 192.168.5.43
   2276 plots of size: 225.290 TiB on-disk, 225.377 TiBe (effective)
Remote Harvester for IP: 192.168.5.60
   3968 plots of size: 392.771 TiB on-disk, 392.925 TiBe (effective)
Remote Harvester for IP: 192.168.5.41
   3352 plots of size: 331.800 TiB on-disk, 331.927 TiBe (effective)
Remote Harvester for IP: 192.168.5.58
   4310 plots of size: 426.616 TiB on-disk, 426.791 TiBe (effective)
Remote Harvester for IP: 192.168.5.54
   2189 plots of size: 216.677 TiB on-disk, 216.762 TiBe (effective)
Remote Harvester for IP: 192.168.5.59
   2974 plots of size: 294.378 TiB on-disk, 294.496 TiBe (effective)
Plot count for all harvesters: 40113
Total size of plots: 3.877 PiB, 3.879 PiBe (effective)
Estimated network space: 31.328 EiB
Expected time to win: 1 day and 19 hours
Note: log into your key using 'chia wallet show' to see rewards for each key

[debug.log](https://github.com/Chia-Network/chia-blockchain/files/13946440/debug.log)

**Comment by BrandtH22** [comment]
Hey @Thanatos-JFJ , as a heads up we can generally provide more timely and thorough support in our Discord server (https://discord.gg/chia). 

I would start by having a new config file created on the farmer:
```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run `chia init`
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Then you may need to have new configs created for the harvesters as well (note- if you do, make sure to add the farmers IP back into the harvester section of the config on each harvester).

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
